src/Compilers/Test/Core/AssemblyLoadTestFixture.cs

```diff
@@ -327,16 +389,28 @@ public int GetFileAttributes(string path)
 
         private static TempFile GenerateDll(string assemblyName, TempDirectory directory, string csSource, params MetadataReference[] additionalReferences)
         {
+            return GenerateDll(assemblyName, directory, csSource, additionalReferences, publicSign: false);
+        }
+
+        private static TempFile GenerateDll(string assemblyName, TempDirectory directory, string csSource, MetadataReference[] additionalReferences, bool publicSign)
```

Comment:
Should 'publicSign' be before 'additionalReferences' here so 'additionalReferences' can still be 'params'?

Reply:
Oh, hmm, it does mean we then need to specify it for everything, but maybe that's for the best to be explicit!

Reply:
> it does mean we then need to specify it for everything

I don't think I understood this, but given this, the existing way is fine, and if you happen to find a way of adjusting the signature that makes it feel nicer that's great too.

Reply:
Oh I see what you mean, keep the params but make the bool go first. Yeah that's better.